src/dsa/rpo_falcon512/error.rs

```diff
@@ -1,6 +1,5 @@
-use core::fmt;
-
 use super::{LOG_N, MODULUS, PK_LEN};
+use core::fmt;
 
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub enum FalconError {
```

Comment:
Are we still using `FalconError` anywhere? If not, I think we should remove it.

Also, we might be able to get rid of the `FalconSerializationError` if we refactor the compress/decompress methods as I mentioned in one of the above comments.

Reply:
Removed and updated accordingly